packages/appkit-utils/src/bitcoin/BitcoinTypesUtil.ts

```diff
@@ -87,6 +94,21 @@ export namespace BitcoinConnector {
        * Specifies which part(s) of the transaction the signature commits to
        */
       sighashTypes: number[]
+      /**
+       * (Optional) The public key corresponding to the private key to use for signing.
+       * At least specify either an address or a public key.
```

Comment:
an address is valid as well here?

Reply:
Not here but we have `address` prop on top (signInputs.address), which here we let users know pass any of them